tests/conftest.py

```diff
@@ -139,6 +138,12 @@ def reporter(request, env):
 
 @pytest.fixture(scope='function')
 def os_user(monkeypatch):
+    if os.name == 'nt':
+        import getpass
```

Comment:
Please import getpass and pwd at the top of the file with the standard lib imports.